cmd/pctl/add.go

```diff
@@ -245,6 +250,25 @@ func getGitRepositoryNamespaceAndName(c *cli.Context) (string, string, error) {
 	return "", "", fmt.Errorf("flux git repository not provided, please provide the --git-repository flag or use the pctl bootstrap functionality")
 }
 
+// getOutFolder returns the output folder with the following precedence:
+// User set --out overrides local configuration.
+// Local configuration, if set.
+// Default out which is `.`.
+func getOutFolder(c *cli.Context) (string, error) {
```

Comment:
```suggestion
func getProfileOutputDirectory(c *cli.Context) (string, error) {
```

I think we try to use `dir/directory` instead of `folder` in general. Also think `out` is a bit vague, `outputDirectory`? or `profileInstallationDirectory`, something a bit more descriptive might be nice

Reply:
Fair point